data-exporter/src/test/java/org/reactome/release/dataexport/EuropePMCTest.java

```diff
@@ -16,6 +16,94 @@
 import static org.reactome.release.dataexport.ReactomeURLConstants.CONTENT_DETAIL_URL;
 
 public class EuropePMCTest {
+	private static EuropePMC europePMC;
+	private static EuropePMC.EuropePMCLink europePMCLink;
+
+	@BeforeAll
+	public static void createEuropePMCInstance() {
+		final String DUMMY_OUTPUT_DIR = "output_dir";
+		final int DUMMY_RELEASE_NUMBER = 999;
+
+		europePMC = EuropePMC.getInstance(DUMMY_OUTPUT_DIR, DUMMY_RELEASE_NUMBER);
+	}
+
+	@BeforeAll
+	public static void createEuropePMCLink() {
+		final String DUMMY_PATHWAY_DISPLAY_NAME = "Cell Cycle";
+		final String DUMMY_PATHWAY_STABLE_ID = "R-HSA-1640170";
+		final String DUMMY_PUBMED_IDENTIFIER = "9153395";
+
+		europePMCLink = new EuropePMC.EuropePMCLink(
+			DUMMY_PATHWAY_DISPLAY_NAME,
+			DUMMY_PATHWAY_STABLE_ID,
+			DUMMY_PUBMED_IDENTIFIER
+		);
+	}
+
+	@Test
+	public void properEuropePMCProfileXML() {
+		final String REACTOME_DESCRIPTION =
+			"Reactome is a free, open-source, curated and peer-reviewed pathway database.";
+
+		final String EXPECTED_EUROPE_PMC_PROFILE_XML = String.join(System.lineSeparator(),
+			"<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>",
```

Comment:
Is it necessary to have XML inline in the test method? Could it be moved to a separate file?

Reply:
I'm kind of inclined to have it remain in the test file just so the test is self-contained and readable at a glance without needing to look up the contents of another file.